cli.js

```diff
@@ -104,7 +104,7 @@ const input = (argv.info || argv.descriptions
 ).filter(
 	// filter excluded scripts
 	i =>
-		!argv.exclude || !argv.exclude.some(e => new RegExp(e + (e.includes('*') ? '' : '$'), 'i').test(i))
+		!argv.exclude || !argv.exclude.some(e => new RegExp(e + (e.includes('*') ? '' : '$'), 'i').test(argv.info || argv.descriptions ? i.value : i))
```

Comment:
it'd be nice to make a refactor later to break this loooooooong line into smaller blocks for readability 😊

Reply:
Jop. Absolutely 🙈